packages/perseus/src/types.ts

```diff
@@ -542,6 +543,11 @@ export type WidgetScorerFunction = (
     locale?: string,
 ) => PerseusScore;
 
+/**
+ * A union type of all the functions to get the public options for each widget.
+ */
+export type SplitWidgetOptionsFunctions = typeof splitCategorizerWidgetOptions;
```

Comment:
Naming: as we're thinking more about what these functions will do, I think they're more about removing than splitting. I think in recent discussions we considered that we might only need to provide a widget options with only public data and then provide the full item for scoring and other server-side work. 

What do you think of simply naming these functions `removePrivateWidgetOptions`, and so this type would be `removePrivateWidgetOptionsFunction` (I think we want this to be singular because it is a union of all the functions, but it represents a single instance of any of them - sorry, it's nitpicky but plural, to me, speaks of an array).

Reply:
For sure. It started out as `PublicWidgetOptionsFunctions` but changed it to "split" after Matthew's feedback. I kind of think it should be in sync with the rest of the naming for this update, which would mean all the other aspects would also change to `removePrivateWidgetOptions`. I do agree that it's not really splitting anymore though. I'm totally cool changing it to `RemovePrivateWidgetOptionsFunction`,  but then I might want to change others like `removePrivateWidgetOptions` as the name on the export and `removeCategorizerPrivateWidgetOptions`. What do you think?

Reply:
I agree.  I think we should be as consistent as possible.